framework/Optimizers/GeneticAlgorithm.py

```diff
@@ -392,6 +392,8 @@ def _useRealization(self, info, rlz1):
     """
     # The whole skeleton should be here, this should be calling all classes and _private methods.
     traj = info['traj']
```

Comment:
how many trajectories are at this point?

Reply:
Our current GA doesn't allow the notion of resistance and hence we only have one trajectory, but we left the multiple trajectories as a hack to allow having a population of individuals not a single individual for the user to input as an initial solution.